src/Components/FilterCard/FilterCard.js

```diff
@@ -32,13 +34,15 @@ export default function FilterCard(){
           <legend>Preferences</legend>
           <label>
             <input
+            className={styles.radioBtn}
               type="checkbox"
               onClick={() => dispatch({ type: "OUTOFSTOCK" })}
             />
             inclue out of stock products
```

Comment:
a typo error here. "include"

Reply:
Thanks for pointing it out I Have changed it